types/config.d.ts

```diff
@@ -26,15 +26,6 @@ export interface IConfig {
     // If set to 'true', clarity-js will NOT write CID (Clarity ID) cookie
     disableCookie?: boolean;
 
-    // If set to false, text on the page will be masked with asterisks
-    showText?: boolean;
-
-    // If set to false, links on the page will be masked with asterisks
-    showLinks?: boolean;
```

Comment:
>showLinks [](start = 4, length = 9)

Is this always done now?

Reply:
Show links was a lousy option, which didn't really work out that great. There is no more special treatment for links. If webmaster wants to see his links, he'd have to set an attribute on them.

---
In reply to: [260855408](https://github.com/Microsoft/clarity-js/pull/137#discussion_r260855408) [](ancestors = 260855408)